### ISSUE TITLE
Crash in GodotPaymentV3.java:71 (NullPointerException)

### ISSUE THREAD

**Issue by mrimvo** [godotengine/godot#38134]
**Godot version:**
Version 3.2.1 stable

**OS/device including version:**
Huawei HUAWEI Y9 Prime 2019 (HWSTK-HF), 3840MB RAM, Android 10

**Issue description:**
Received this bug report from one of my app users.

```
java.lang.NullPointerException: 
  at org.godotengine.godot.GodotPaymentV3.<init> (GodotPaymentV3.java:71)
  at org.godotengine.godot.GodotPaymentV3.initialize (GodotPaymentV3.java:63)
  at org.godotengine.godot.GodotLib.setup (Native Method)
  at org.godotengine.godot.Godot$2.run (Godot.java:325)
  at android.opengl.GLSurfaceView$GLThread.guardedRun (GLSurfaceView.java:1511)
  at android.opengl.GLSurfaceView$GLThread.run (GLSurfaceView.java:1281)
```

**Comment by akien-mga** [comment]
CC @m4gr3d 

This might have been improved already by refactoring work in 3.2.2 beta 1 if you are able to test: https://godotengine.org/article/dev-snapshot-godot-3-2-2-beta-1

**Comment by mrimvo** [comment]
I had a look at the code for both 3.2.1 stable tag and the 3.2 branch. I'm pretty sure it is solved in 3.2.2:

old:
```
	public GodotPaymentV3(Activity p_activity) {

		registerClass("GodotPayments", new String[] { "purchase", "setPurchaseCallbackId", "setPurchaseValidationUrlPrefix", "setTransactionId", "getSignature", "consumeUnconsumedPurchases", "requestPurchased", "setAutoConsume", "consume", "querySkuDetails", "isConnected" });
		activity = (Godot)p_activity;
		mPaymentManager = activity.getPaymentsManager();
		mPaymentManager.setBaseSingleton(this); // <--- crash here (NPE)
	}

new:
```
	public GodotPayment(Godot godot) {
		super(godot);
		mPaymentManager = new PaymentsManager(godot, this);
		mPaymentManager.initService(); // no NPE here, but maybe downstream
	}
```
